core/src/main/resources/application.properties

```diff
@@ -168,8 +171,19 @@ spring.jpa.hibernate.jdbc.lob.non_contextual_creation=true
 spring.jpa.properties.hibernate.naming.physical-strategy=org.hibernate.boot.model.naming.PhysicalNamingStrategyStandardImpl
 
 # Azure, Google OAuth2 login url
-klaw.sso.server.loginurl.azure=/oauth2/authorization/azure
-klaw.sso.server.loginurl.google=/oauth2/authorization/google
+klaw.sso.server.loginurl.azure.url=/oauth2/authorization/azure
+klaw.sso.server.loginurl.google.url=/oauth2/authorization/google
+#klaw.sso.server.loginurl.github.url=/oauth2/authorization/github
+
+klaw.sso.server.loginurl.azure.imageURI=assets/images/clients/azuread.svg
+klaw.sso.server.loginurl.google.imageURI=assets/images/clients/google.svg
+#klaw.sso.server.loginurl.github.imageURI=
+
+klaw.sso.server.loginurl.azure.ssoProvider=Azure
```

Comment:
Do we need these ? Can we get the providers from  defined login urls ? as there we have providers mentioned already.
Ex : klaw.sso.server.provider.**azure**.loginurl=/oauth2/authorization/azure

Reply:
this way, we can enforce a login url

Reply:
Or depending on the configs like below, we know which are configured if they are not null or commented
spring.security.oauth2.client.registration.**azure**.client-id

And I think login urls are always fixed. /oauth2/authorization/**providername**. Can we remove these login urls config totally and just have one config baseLoginUrl = /oauth2/authorization 

Just thinking if these are any possibilities

Reply:
I might be able to make that change will give it a quick look

Reply:
So I can 100% remove the ssoProvider, I can also remove the login url however if i do that, I'll have to make a larger change. as then I'll have to pull it from the active security registration config and leave Google and Azure hardcoded in the html as we want to leave those there.